.changeset/cold-cooks-battle.md

```diff
@@ -0,0 +1,21 @@
+---
+'@astrojs/node': minor
+---
+
+Allows disabling Astro's default [HTML streaming](https://docs.astro.build/en/guides/on-demand-rendering/#html-streaming) for pages rendered on demand
+
+HTML streaming helps with performance and generally provides a better visitor experience. In most cases, disabling streaming is not recommended.
+
+However, when you need to disable HTML streaming (e.g. your host only supports non-streamed HTML caching at the CDN level), you can opt out of the default behavior:
```

Comment:
```suggestion
However, when you need to disable HTML streaming (e.g. your host only supports non-streamed HTML caching at the CDN level), you can now opt out of the default behavior:
```
minor nit, but I like this for the changelog announcement to reinforce it's new and different!